dev/packages/example/base-1.0.0/elasticsearch/component-template/events-mappings.json

```diff
@@ -0,0 +1,148 @@
+{
```

Comment:
More general question we had around `ingest-pipeline` directory. Should it be `component_template` or `component-template`? What is our rule? Any preference?

Reply:
I have no opinion here, I followed the example of `ingest-pipeline`.

(Kibana saved objects use hyphenation and not snake case, maybe that's how the initial format was derived even for assets that are not Kibana assets.)

Reply:
lets keep the `-` for now. The part where I stumbled over `_` is in the ES API.